### DISCUSSION TITLE
404 Not found after refresh page

### DISCUSSION THREAD

**Discussion by metilnodes** [ping-pub/explorer#213] [Q&A]
Hey, i create dashboard http://94.130.200.205/
but when i go to network (any page) and then refresh page - i have an error 
404 Not Found
nginx/1.18.0 (Ubuntu)

so i need back to address: http://94.130.200.205/ and start from beginning - can you help with that how to fix?

**Comment by liangping** [comment]
``` nginx
   location / {
        root /usr/share/nginx/html;
        index  index.html index.htm;
        try_files $uri $uri/ /index.html;  # add this line to your config
    }
```

**Reply by metilnodes** [reply]
Hey, hm, this line added default =/

**Reply by metilnodes** [reply]
<img width="792" alt="Screen Shot 2022-10-01 at 04 52 35" src="https://user-images.githubusercontent.com/83473551/193378513-590be3d7-dfa2-4570-aa22-55482aa7475f.png">

**Reply by liangping** [reply]
That's all you need

**Reply by metilnodes** [reply]
but still have this error...

**Reply by liangping** [reply]
replace server name 'HAQQ' to '_'

**Reply by metilnodes** [reply]
Changed back - rebuild etc - same problem ;(
Update in any pages - 404 error

**Reply by metilnodes** [reply]
reinstalled all, nothing help, same situation ;( real don't understand what's wrong

**Reply by metilnodes** [reply]
Hey boss, sry for disturb again. Still have this issue ;(
